include/cantera/oneD/MultiNewton.h

```diff
@@ -85,4 +85,4 @@ class MultiNewton
 };
 }
 
-#endif
+#endif
```

Comment:
Please undo this. We like our files to have a newline character on the last line (but not a blank line after that).

Reply:
What exactly needs to undone here? An extra line added after the endif?

Reply:
Ah, the comment didn't pick this up clearly. One of your commits removed the end-of-file newline character here.